.github/workflows/on-push.yml

```diff
@@ -0,0 +1,22 @@
+name: Run Tests and Linting
+
+on:
+  push
+
+jobs:
+  deploy:
+    name: Run Tests and Linting
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Ref
+        uses: actions/checkout@v1
+        with: 
+          ref: ${{ github.event.push.after }}
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '13'
+      - name: Run Tests and Lint
+        run: bash -e ./scripts/test.sh
```

Comment:
Same comment as before about not adding another layer of indirection by going to the _./scripts/test.sh_ file. Just put the commands inline

```
- name: Install dependencies
  run: |
    npm ci
- name: Run tests and Lint
  run: |
    npm test
    npm run build
```